core/server/api/v2/utils/serializers/input/pages.js

```diff
@@ -8,24 +8,27 @@ function removeMobiledocFormat(frame) {
     }
 }
 
+/**
+ * CASE:
+ *
+ * - the content api endpoints for pages forces the model layer to return static pages only
+ * - we have to enforce the filter
+ *
+ * @TODO: https://github.com/TryGhost/Ghost/issues/10268
+ */
+function addPageFilter(frame) {
+    if (frame.options.filter) {
+        frame.options.filter = `${frame.options.filter}+page:true`;
+    } else {
+        frame.options.filter = 'page:true';
+    }
+}
+
 module.exports = {
     browse(apiConfig, frame) {
         debug('browse');
 
-        /**
-         * CASE:
-         *
-         * - the content api endpoints for pages forces the model layer to return static pages only
-         * - we have to enforce the filter
-         *
-         * @TODO: https://github.com/TryGhost/Ghost/issues/10268
-         */
-        if (frame.options.filter) {
-            frame.options.filter = `${frame.options.filter}+page:true`;
-        } else {
-            frame.options.filter = 'page:true';
-        }
-
+        addPageFilter(frame);
```

Comment:
Now that this method is only used once. Let's just keep it as was, so the PR is just focused on one specific change? ;)